packages/chatbot-server-mongodb-public/src/mongoDbMetadata/products.ts

```diff
@@ -0,0 +1,320 @@
+import { z } from "zod";
+import { mongoDbProgrammingLanguageIds } from "./programmingLanguages";
+
+export const MongoDbProductSchema = z.object({
+  id: z.string().describe("Unique identifier for the product"),
+  name: z.string().describe("Human-friendly name of the product"),
+  description: z
+    .string()
+    .optional()
+    .describe("Brief description of the product"),
+  programmingLanguage: z
+    .enum(mongoDbProgrammingLanguageIds)
+    .optional()
+    .describe("Programming language"),
```

Comment:
Can we this description mention the product too? e.g. "The programming language used to interact with the product"

Reply:
```suggestion
    .describe("The programming language used to interact with the product"),
```